config/config_test.go

```diff
@@ -130,6 +130,18 @@ func TestDecodeConfig(t *testing.T) {
 		},
 		config.Service.Pipelines["traces"],
 		"Did not load pipeline config correctly")
+
+	assert.Equal(t,
+		&configmodels.Pipeline{
+			Name:       "traces/second-stage-exporters",
```

Comment:
Is the "second stage" term documented anywhere? Seems too generic, so I think we need clear semantics / example.

Reply:
Not that I'm aware. First time I saw it was here: https://github.com/open-telemetry/opentelemetry-collector/issues/1260#issuecomment-663152362